gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/auth/GcsDelegationTokens.java

```diff
@@ -66,7 +75,8 @@ public GcsDelegationTokens() throws IOException {
   @Override
   public void serviceInit(Configuration conf) {
     String tokenBindingImpl = DELEGATION_TOKEN_BINDING_CLASS.get(conf, conf::get);
-
+    this.tokenInstantiationStrategy =
```

Comment:
`this.` prefix usually used only in constructors/setters when there is a parameter with the same name.

Reply:
done